Frontend_Web_Development_React_Angular_Vue/React/Week 2/Components.md

```diff
@@ -0,0 +1,110 @@
+<div align="center">
+    <img src="https://cdn.educba.com/academy/wp-content/uploads/2020/03/React-Components.jpg" alt="react-components" height="80%" width="80%">
+</div>
+<br>
+<div align="center">
+    <h1> What is Component? </h1> 
+</div> 
+ 
+## Introduction
+
+Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. In this lesson, we'll learn introduction to components in React and how to use them in our React applications and what are the types of it.
+
+Okay so let's start with an example. Let's write the simplest React component:
+
+```js
+const App = () => {
+    return (
+        <h1>Hello World!</h1>
+    );
+};
+
+ReactDOM.render(<App />, document.getElementById("root"));
+```
+This is a simplest React Application anyone can write, let's understand it line by line:
+- The first thing we do here, is to make our own component, `App`. React is all about making components. And then taking those components and making more components out of those.
```

Comment:
We don't make components out of every single component. We just do it for the ones where reusability or modularity is possible. Please add this.